### DISCUSSION TITLE
Game struggles to run very large beatmaps

### DISCUSSION THREAD

**Discussion by ziv-vy** [ppy/osu#29705] [Q&A] [answered]
### Type

Performance

### Bug description

The game struggles to load very big maps.

### Screenshots or videos

https://youtu.be/SdH5ObaBnZE

### Version

2024.817.0-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16888315/compressed-logs.zip)

**Comment by peppy** [comment]
Please link the beatmap.

**Reply by ziv-vy** [reply]
https://drive.google.com/file/d/108kYJ-Gi3Sn89jCeIGqpkSjCpce562Uq/view

**Comment by smoogipoo** [comment, marked as answer]
This is not a supported scenario.
